packages/framework/src/Framework/Actions/GeneratesDocumentationSearchIndex.php

```diff
@@ -48,7 +59,7 @@ protected function __construct()
     protected function run(): void
     {
         DocumentationPage::all()->each(function (DocumentationPage $page): void {
-            if (! in_array($page->identifier, Config::getArray('docs.exclude_from_search', []))) {
+            if (! in_array($page->identifier, array_merge(Config::getArray('docs.exclude_from_search', []), ['search']))) {
```

Comment:
Could cause issues if the user wants to document their search page, maybe we tie it into the create_search_page config option? Because if they have a custom search page, they would want to disable that feature, and then this should also be disabled.

Reply:
See 95dfaf14246d85b6de754a7618e5f1f968473c37